tests/utils/test_process_utils.py

```diff
@@ -97,6 +97,10 @@ def test_reap_process_group(self):
 
 
 class TestExecuteInSubProcess(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def _tmp_path(self, tmp_path):
+        self.tmp_path = tmp_path
```

Comment:
Why is this added to all tests in this class? I guess this will only be used in one method so we can do it at method level.

Reply:
Removed